src/tests/dbus-tests/udiskstestcase.py

```diff
@@ -37,7 +37,8 @@ def run_command(command):
 def get_call_long(call):
     def call_long(*args, **kwargs):
         """Do an async call with a very long timeout (unless specified otherwise)"""
-        kwargs['timeout'] = 100  # seconds
+        if not 'timeout' in kwargs:
```

Comment:
nitpick: `if 'timeout' not in kwargs:` would look better :-)

Reply:
Ah! Fixed.